### PR TITLE
Convert card refactor

### PR DESCRIPTION
**Pull request by Tshamp7**
Moved ConvertCard to its own file and imported into index.txs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,6 +62,7 @@
     "@types/styled-components": "^5.1.3",
     "react-autosize-textarea": "^7.1.0",
     "react-copy-to-clipboard": "^5.0.2",
+    "react-in-out-textarea": "^0.0.3",
```
Comment (igeligel) [inline package.json:65]: I think you do not have to include that here. Why was this added?

File: src/styled/ConvertCard.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+const ConvertCard = styled.div`
```
Comment (igeligel) [inline src/styled/ConvertCard.ts:3]: Can we use a named export here? You can read about it here: https://basarat.gitbook.io/typescript/main-1/defaultisbad

**Review by igeligel — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Tshamp7) [reply package.json:65]: Im not sure why my system added that. Ill fix it!

Comment (Tshamp7) [reply package.json]: Hmm I am not sure why yarn added that. I will fix it.

File: src/styled/ConvertCard.ts (hunk 2 above)
Comment (Tshamp7) [reply src/styled/ConvertCard.ts]: sure thing!

**Comment by igeligel** [comment]
Ok one last thing that I actually forgot to mention. Can you delete the `.DS_Store` file? Then I will merge that Pull Request 👍

**Comment by igeligel** [comment]
I think you need to first delete the file manually and then add it to the gitignore. But feel free to leave the gitignore change out.
